plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -79,6 +79,11 @@ describe('PersonState.update()', () => {
         return (await hub.db.clickhouseQuery(query)).data
     }
 
+    async function fetchPersonsRowsWithVersionHigerEqualThan(version = 1) {
```

Comment:
Do we specifically want to get the versions higher than the specified `version` or are we actually looking for the latest version, and we want to wait until it matches certain criteria?

Reply:
specifically higher than - I want to get version = 1 (i.e. merged person) & version = 100 (i.e. deleted person)

Reply:
Where does 100 come from BTW? I don't see it in the test aside from the assertion.

Reply:
it's a magic number atm but should make it a const as Yakko has called out too: https://github.com/PostHog/posthog/blob/master/posthog/models/person/util.py#L182

Reply:
the reasoning behind it being + 100 rather than +1 is that if there was some parallel processing, then the +100 wins to be latest